docs/src/main/sphinx/connector/filesystem-cache.md

```diff
@@ -0,0 +1,114 @@
+# Filesystem cache
+
+Trino accesses files directly on object storage and remote filesystem storage.
+This often involves the transfer of large amounts of data. The files are
+retrieved from HDFS, or any other supported object storage, by multiple workers
+and processed on these workers. Repeated queries with different parameters, or
+even different queries from different users, often access, and therefore
+transfer, the same objects.
+
+Trino includes support for caching these files with the help of the open
+source [Alluxio](https://github.com/Alluxio/alluxio) libraries with catalogs
+using the following connectors:
+
+* [](/connector/delta-lake)
+
+## Benefits
+
+Enabling caching can result in the following significant benefits:
+
+**Reduced load on storage**
+
+Every retrieved and cached file avoids repeated retrieval from the storage in
+subsequent queries on the same worker. As a result the storage system does not
+have to provide the file again and again.
+
+For example, if your query accesses 100MB of files from the storage, the first
+time the query runs 100MB are downloaded and cached. Any following query uses
+these files. If your users run another 100 queries accessing the same files,
+your storage system does not have to provide all data repeatedly. Without
+caching it has to provide the same files again and again, resulting in up to
+10GB of total files to serve.
+
+**Increased query performance**
+
+Caching can provide significant performance benefits, by avoiding the repeated
+network transfers and instead accessing copies of the files from a local
+cache. Performance gains are more significant if the performance of directly
+accessing the storage is low compared to accessing the local cache.
+
+For example, if you access storage in a different network, different data
+center, or even different cloud-provider region query performance is slow. Adding
+caching using fast, local storage has a significant impact and makes your
+queries much faster.
+
+On the other hand, if your storage is already running at very high performance
+for I/O and network access, and your local cache storage is at similar speeds,
+or even slower, performance benefits can be minimal.
+
+**Reduced query costs**
+
+A result of the reduced load on the storage, mentioned earlier, is significantly
+reduced network traffic. Network traffic is often a considerable cost factor,
```

Comment:
This is a nitpick: The reduced cost is usually not because of reduced network traffic, which is usually free within a data centre, but the reduction in API operations against object storage, for which the cloud providers charge a lot.

Reply:
I reworded so it is both .. since different platforms and setups have different cost structures